### PR TITLE
Added Keymap, removed RGB animations, added custom font for 0xCB 1337 

### PR DESCRIPTION
**Pull request by Conor-Burns**
Signed-off-by: Conor Burns <mail@conor-burns.com>

## Description

- added the keymap "jakob"
- made a custom font
- changed some OLED code for clearing the display (thanks for all the help in the discord :))
- added RGB mode display for a specific layer
- flipped encoder

## Types of Changes

- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)

## Issues Fixed or Closed by This PR

- only stuff from my board :D

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/0xcb/1337/config.h [hunk 1]
```diff
@@ -17,6 +17,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#ifdef RGBLIGHT_ENABLE
+  #include <stdio.h>
+#endif
```
Comment (fauxpark) [inline keyboards/0xcb/1337/config.h:22]:
```suggestion
```
This should not be needed.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/0xcb/1337/config.h (hunk 1 above)
Comment (Conor-Burns) [reply keyboards/0xcb/1337/config.h:22]: just wanted to minimize the firmware when RGB and thus the RGB modes display is deactivated, will of course remove this logic if you deem it unnecessary

Comment (fauxpark) [reply keyboards/0xcb/1337/config.h:22]: stdio.h is indirectly included regardless of whether `RGBLIGHT_ENABLE` is defined, and generally header file inclusions don't/shouldn't affect filesize.

**Comment by Conor-Burns** [comment]
@fauxpark Could you rerun Travis CI? it exceeded the time limit?!

**Comment by fauxpark** [comment]
You'll need to rebase the PR so that Travis only picks up these changes.

**Comment by Conor-Burns** [comment]
Ah thanks - done

File: keyboards/0xcb/1337/config.h [hunk 2]
```diff
@@ -17,6 +17,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 #include "config_common.h"
+#include <stdio.h>
```
Comment (fauxpark) [inline keyboards/0xcb/1337/config.h:20]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/0xcb/1337/config.h (hunk 2 above)
Comment (Conor-Burns) [reply keyboards/0xcb/1337/config.h]: I've now moved it to the keymap - if I omit it, it does not compile https://s.0xcb.dev/95oKo

**Review by fauxpark — APPROVED** [review]

File: keyboards/0xcb/1337/gfxfont.c [hunk 3]
```diff
@@ -0,0 +1,260 @@
+#include "progmem.h"
```
Comment (drashna) [inline keyboards/0xcb/1337/gfxfont.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/0xcb/1337/config.h [hunk 4]
```diff
@@ -65,6 +70,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LOCKING_RESYNC_ENABLE
 /* Oled Size */
 #define OLED_DISPLAY_128X64
+#define OLED_FONT_END 255
```
Comment (drashna) [inline keyboards/0xcb/1337/config.h:73]: Nice. :)

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/0xcb/1337/config.h (hunk 4 above)
Comment (Conor-Burns) [reply keyboards/0xcb/1337/config.h:73]: huh?

File: keyboards/0xcb/1337/gfxfont.c (hunk 3 above)
Comment (Conor-Burns) [reply keyboards/0xcb/1337/gfxfont.c:17]: ah sry makes sense as its a .c file

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**